groups.rst

```diff
@@ -1,216 +1,218 @@
+.. Do NOT modify this file directly; edit summary.yaml instead.
+
 International In-Kind Contribution Program
 ------------------------------------------
 
 
-**ESP-BCM-S2:** Dr. Ignacio Sevilla-Noarbe
+**ESP-BCM-S2:** *Technical support in commissioning*
 
-*Technical Support in Commissioning*
+  Point of Contact: Ignacio Sevilla-Noarbe
 
-Ignacio Sevilla, Cristobal Padilla, Otger Ballester, Alexandre Pérez, Jordi Arellano, Ricard Casas
+  Members: Ignacio Sevilla, Cristobal Padilla, Otger Ballester, Alexandre Pérez, Jordi Arellano, Ricard Casas
 
 
-**FRA-INP-S6:** Dr. Jérémy Neveu
+**FRA-INP-S6:** *Calibration hardware (CBP, flat screen, dome illumination, etc.)*
 
-*Calibration hardware (CBP, flat screen, dome illumination, etc.)*
+  Point of Contact: Jérémy Neveu
 
-Jérémy Neveu, Laurent Le Guillou, Eduardo Sepulveda, Sylvain Baumont
+  Members: Jérémy Neveu, Laurent Le Guillou, Eduardo Sepulveda, Sylvain Baumont
 
 
-**FRA-INP-S7:** Dr. Marc Moniez
+**FRA-INP-S7:** *AuxTel support*
 
-*AuxTel support*
+  Point of Contact: Marc Moniez
 
-Marc Moniez, Sylvie Dagoret-Campagne, Martin Monroy, Jérémy Neveu, Laurent Le Guillou
+  Members: Marc Moniez, Sylvie Dagoret-Campagne, Martin Monroy, Jérémy Neveu, Laurent Le Guillou
 
 
-**FRA-INP-S8:** Dr. Pierre Antilogus
+**FRA-INP-S8:** *Commissioning at pixel level, data quality and “on-sky” calibration*
 
-*Commissioning at pixel level, data quality and “On-Sky” calibration*
+  Point of Contact: Pierre Antilogus
 
-Dominique Boutigny, Sabine Elles, Thibault Guillemin, Fabio Hernandez, Fabrice Feinstein, Dominique Fouchez, Benjamin Racine, André Tilquin, Emmanuel Gangler, Fabrice Jammes, Nicoleta Pauna, Pierre Antilogus, Pierre Astier, Claire Juramy, Aurélien Barrau, Johan Brégeon, Céline Combet, Marine Kuna, Johann Cohen-Tanugi
+  Members: Dominique Boutigny, Sabine Elles, Thibault Guillemin, Fabio Hernandez, Fabrice Feinstein, Dominique Fouchez, Benjamin Racine, André Tilquin, Emmanuel Gangler, Fabrice Jammes, Nicoleta Pauna, Pierre Antilogus, Pierre Astier, Claire Juramy, Aurélien Barrau, Johan Brégeon, Céline Combet, Marine Kuna, Johann Cohen-Tanugi
 
 
-**FRA-INP-S9:** Dr. Cyrille Doux
+**FRA-INP-S9:** *Object detection, weak lensing, and deblending*
 
-*Object detection, weak lensing, and deblending*
+  Point of Contact: Cyrille Doux
 
-Eric Aubourg, Cécile Roucelle, Alexandre Boucaud, Justine Zeghal, Biswajit Biswas, Axel Guinot, Marina Ricci
+  Members: Eric Aubourg, Cécile Roucelle, Alexandre Boucaud, Justine Zeghal, Biswajit Biswas, Axel Guinot, Marina Ricci
 
 
-**FRA-INP-S10:** Dr. Alexandre Boucaud
+**FRA-INP-S10:** *Camera hardware optimization*
 
-*Camera hardware optimization*
+  Point of Contact: Alexandre Boucaud
 
-Eric Aubourg, Cécile Roucelle, Alexandre Boucaud, Justine Zeghal, Biswajit Biswas, Axel Guinot, Marina Ricci
+  Members: Eric Aubourg, Cécile Roucelle, Alexandre Boucaud, Justine Zeghal, Biswajit Biswas, Axel Guinot, Marina Ricci
```

Comment:
Hi Keith, two comments regarding that contribution.
1. After discussing this with Pierre Antilogus a while ago, I thought we had agreed this contribution would be included in the broader camera optimization effort rather than a separate one.
2. In any case, such task would only be performed by Eric Aubourg and I, not the other IN2P3-APC people listed.

ping @aubourg

Reply:
Updated list of names as suggested